### ISSUE TITLE
Unusual loss graph

### ISSUE THREAD

**Issue by DilaraAlbayrak** [RBirkeland/MVCNN-PyTorch#12]
Normally, loss graphs have this kind of decreasing/increasing trend: 

![losses](https://user-images.githubusercontent.com/7215154/66915458-bbfcba80-f021-11e9-9d64-30cd60dada35.png)

However, with this implementation on my dataset i got graphs like below:

![loss](https://user-images.githubusercontent.com/7215154/66915504-d767c580-f021-11e9-817c-d24eff2a731b.png)

Why this might be the case and how can i solve this?

**Comment by RBirkeland** [comment]
I would guess overfitting to the training set.

**Comment by DilaraAlbayrak** [comment]
Thx, i will try some variations in dataset
